generators/lua/lua_generator.ts

```diff
@@ -70,38 +77,39 @@ export class LuaGenerator extends CodeGenerator {
-      'printError,rawequal,rawget,rawset,read,rednet,redstone,rs,select,' +
-      'setfenv,setmetatable,sleep,string,table,term,textutils,tonumber,' +
-      'tostring,turtle,type,unpack,vector,write,xpcall,_VERSION,__indext,' +
-      // Not included in the script, probably because it wasn't enabled:
-      'HTTP,' +
-      // Keywords (http://www.lua.org/pil/1.3.html).
-      'and,break,do,else,elseif,end,false,for,function,if,in,local,nil,not,' +
-      'or,repeat,return,then,true,until,while,' +
-      // Metamethods (http://www.lua.org/manual/5.2/manual.html).
-      'add,sub,mul,div,mod,pow,unm,concat,len,eq,lt,le,index,newindex,call,' +
-      // Basic functions (http://www.lua.org/manual/5.2/manual.html,
-      // section 6.1).
-      'assert,collectgarbage,dofile,error,_G,getmetatable,inpairs,load,' +
-      'loadfile,next,pairs,pcall,print,rawequal,rawget,rawlen,rawset,select,' +
-      'setmetatable,tonumber,tostring,type,_VERSION,xpcall,' +
-      // Modules (http://www.lua.org/manual/5.2/manual.html, section 6.3).
-      'require,package,string,table,math,bit32,io,file,os,debug'
+        // From theoriginalbit's script:
+        // https://github.com/espertus/blockly-lua/issues/6
+        '__inext,assert,bit,colors,colours,coroutine,disk,dofile,error,fs,' +
+        'fetfenv,getmetatable,gps,help,io,ipairs,keys,loadfile,loadstring,math,' +
+        'native,next,os,paintutils,pairs,parallel,pcall,peripheral,print,' +
+        'printError,rawequal,rawget,rawset,read,rednet,redstone,rs,select,' +
+        'setfenv,setmetatable,sleep,string,table,term,textutils,tonumber,' +
+        'tostring,turtle,type,unpack,vector,write,xpcall,_VERSION,__indext,' +
+        // Not included in the script, probably because it wasn't enabled:
+        'HTTP,' +
+        // Keywords (http://www.lua.org/pil/1.3.html).
+        'and,break,do,else,elseif,end,false,for,function,if,in,local,nil,not,' +
+        'or,repeat,return,then,true,until,while,' +
+        // Metamethods (http://www.lua.org/manual/5.2/manual.html).
+        'add,sub,mul,div,mod,pow,unm,concat,len,eq,lt,le,index,newindex,call,' +
+        // Basic functions (http://www.lua.org/manual/5.2/manual.html,
+        // section 6.1).
+        'assert,collectgarbage,dofile,error,_G,getmetatable,inpairs,load,' +
+        'loadfile,next,pairs,pcall,print,rawequal,rawget,rawlen,rawset,select,' +
+        'setmetatable,tonumber,tostring,type,_VERSION,xpcall,' +
+        // Modules (http://www.lua.org/manual/5.2/manual.html, section 6.3).
+        'require,package,string,table,math,bit32,io,file,os,debug',
     );
   }
 
   /**
    * Initialise the database of variable names.
-   * @param {!Workspace} workspace Workspace to generate code from.
+   *
+   * @param workspace Workspace to generate code from.
    */
-  init(workspace) {
+  init(workspace: Workspace) {
     // Call Blockly.CodeGenerator's init.
-    super.init();
+    super.init(workspace);
```

Comment:
Although `workspace` wasn't passed in previously to `super.init`, functionality should be the same since `CodeGenerator.init` doesn't actually use the param.

Reply:
Yeah, I had to fix this in at least one of the other generators too.